test/test_fake_tensor.py

```diff
@@ -554,8 +554,19 @@ def test_cuda_lstm(self):
     def test_data_dependent_operator(self):
         with FakeTensorMode(allow_fallback_kernels=False):
             x = torch.rand([10, 10])
-
             self.assertRaises(DynamicOutputShapeException, lambda: torch.nonzero(x))
+            self.assertRaises(DynamicOutputShapeException, lambda: torch.unique(x))
+
+        # unique()
+        shape_env = ShapeEnv()
+        with FakeTensorMode(shape_env=shape_env):
+            x = torch.rand([10, 10])
+            y = torch.unique(x)
+            self.assertEqual(type(y.shape[0]), torch.SymInt)
```

Comment:
Please add a couple tests using `torch.ops.aten._unique2.default` and `return_counts` and `return_inverse`

Reply:
added test cases as requested.